target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8065-nighthawk.dtsi

```diff
@@ -0,0 +1,456 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+#include "qcom-ipq8065.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Netgear Nighthawk";
+	compatible = "netgear,r7800","netgear,xr500", "qcom,ipq8065", "qcom,ipq8064";
```

Comment:
I'd drop compatible and model for the DTSI, it will be overwritten in DTS files anyway.

Reply:
dropped